skipper.go

```diff
@@ -1060,3 +1065,21 @@ func Run(o Options) error {
 
 	return listenAndServe(proxy, &o)
 }
+
+func parseAccessLogFilter(accessLogDisabled bool, filter string) al.AccessLogFilter {
```

Comment:
There is already filter parser in `eskip` package, that you can use, why not use this instead of this function?